recovery/elliptics_recovery/types/merge.py

```diff
@@ -619,6 +624,7 @@ def recover(self):
         self.recover_result = Recovery(key=self.id,
                                        timestamp=self.timestamp,
                                        size=self.size,
+                                       flags=0,
```

Comment:
This is method of `DumpRecover` that uses lookups for gathering meta of keys replicas. Lookup command doesn't provide record flags because `dnet_file_info` can't be changed without breaking elliptics protocol. Also for saving behaviour of reading/looking up uncommitted keys, these commands returns `-2` (`-ENOENT`) for such keys. This error can be changed to `-61` (`-ENODATA`) but still we can't provide records flags for recovering.